### PR TITLE
Mobile portfolio

### PR DESCRIPTION
**Pull request by Bushido-brown**
Finished setting up the Mobile version Skeleton of Portfolio.

### CODE REVIEW

File: styles/style.css [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+  color: #6070ff;
+}
+.navigation {
+  width: 18px;
+  height: 12px;
+  left: 3px;
+  top: 6px;
+}
+.container {
+}
```
Comment (vic778) [inline styles/style.css]: kindly delete these lines because there empty

File: index.html [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+        <h1 class="box1">I'm Tochukwu Glad to see you!</h1>
+        <p class="box2">I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+        <p class="box3">LET'S CONNECT</p>
+       
+    </section>
+    <footer class="socials">
+        <img src="/img/social media icons.png">
+    </footer>
+</body>
+</html>
```
Comment (vic778) [inline index.html]: kindly you need to fix your margin and your padding. there is too much space between your footer and the end of the page. please make sure you're following he [Figma](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/images/m1_setup/headline-mobile.png) design. inspect the page to know the padding and the margin of your template in Figma. often it is symbolized by a `*`

**Review by vic778 — CHANGES_REQUESTED** [review]
### STATUS: CHANGES REQUIRED :yellow_circle: 
Hi @Bushido-brown , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

### Project highlights:
- [x] All linter checks are passing :heavy_check_mark: 
- [x] Descriptive commits and correct use of the Git Flow :fire:
- [x] :white_check_mark: page titles :white_check_mark: text headings :white_check_mark: color contrast :white_check_mark: resize :white_check_mark: basic structure of the page 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


so close

Cheers and Happy coding! 👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Kingstalux — COMMENTED** [review]
Hi @Bushido-brown , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.
68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f644550476b6e5269456e6965632f67697068792e676966



## Invalid Code Review Request 🛑 


- [ ] You have submitted a project that does not have the linters checks. Kindly configure your linters correctly and make sure the checks show up on GitHub and they should be passing before requesting for a review 👍 

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: index.html [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+        </nav>
+    </header>
+
+    <section class="container">
+        <h1 class="box1">I'm Tochukwu Glad to see you!</h1>
+        <p class="box2">I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+        <p class="box3">LET'S CONNECT</p>
+    </section>
+    <footer class="socials">
+        <img src="/img/socialmediaicons.png" alt="#">
```
Comment (smunozmo) [inline index.html:25]: [OPTIONAL] This `img` is not working, just correct the file name.

File: index.html [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+    <section class="container">
+        <h1 class="box1">I'm Tochukwu Glad to see you!</h1>
+        <p class="box2">I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+        <p class="box3">LET'S CONNECT</p>
+    </section>
+    <footer class="socials">
+        <img src="/img/socialmediaicons.png" alt="#"> 
+    </footer>
+</body>
+</html>
```
Comment (smunozmo) [inline index.html:28]: - [ ] Great job removing the unncessary code, however, your padding and margin is still looking fairly different from the original design:

Template: 
Captura de pantalla 2022-03-18 a la(s) 9 43 21 p  m

Your design:
Captura de pantalla 2022-03-18 a la(s) 9 43 39 p  m

Please address this problem and you will be done! 💯

**Review by smunozmo — CHANGES_REQUESTED** [review]
Hi @Bushido-brown, 👋  

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights 👏 

- Great work using the right GitHub Flow.
- Your README looks professional, well done!
- Commits are descriptive, good work.

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._




Cheers and Happy coding! 🤓 

✅  Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

⚠️  **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by BenMukebo — APPROVED** [review]
### Hi @Bushido-brown , 👋 

# STATUS: Approved  🎉 
Your project is complete!  There is nothing else to say other than... it's time to merge it
Congratulations! 🎉  🎉 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on( @BenMukebo ) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**